index.html

```diff
@@ -30,7 +30,7 @@ <h3><a href="/"></a>My Logo</h3>
       <section class="bg-color-1">
         <div class="bg-image-1"></div>
         <div class="headline">
-          <h1 class="headline-inside">I’m Daphine Glad to see you!</h1>
+          <h2 class="headline-inside">I’m Daphine Glad to see you!</h2>
           <p class="headline-inside2">
             I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If
             you like what you see and have a project you need coded, don’t hestiate to contact me.
```

Comment:
- [ ] So as I can't select the specific line of codes, I select this part. Just after this part, the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href, please kindly add your `img` tag in the `a` tag 🙃 🙃